string-handling/string-characteristics/c-sharp/change_string.cs

```diff
@@ -0,0 +1,45 @@
+/*
+Isaac Computer Science
+Usage licensed under the Open Government Licence v3.0
+Note: This file is designed to be copied out and compiled on your machine.
+In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file.
+To run this file you need to:
+1. Copy the contents
+2. Paste them into the C# IDE of your choice (Visual Studio, for example)
+3. Change the namespace to match your project (if neccesary)
+4. Compile the program
+5. Run the program
+*/
+
+using System;
+
+namespace IsaacCodeSamples
+{
+
+    class StringCharacteristics
+    {
+
+        // The Main method is the entry point for all C# programs
+        public static void Main() {
+            string myString = "An aardvark is an animal";
+            string newString = ChangeString(myString);
```

Comment:
A bit of a pointless variable - I can see why you did it this way. Not sure what our precedent is with this kind of thing though. It's probably more clear for lower ability students but perhaps doesn't model the best practice. Tricky.

Reply:
We did it for clarity (for weaker students). Maybe we can review whn GCSE content is added.